Discussion/Discussion/Presentation/Comments/Responses/ResponsesViewModel.swift

```diff
@@ -88,19 +90,23 @@ public class ResponsesViewModel: BaseResponsesViewModel, ObservableObject {
     }
     
     @MainActor
-    func getComments(commentID: String, parentComment: Post, page: Int) async -> Bool {
+    func getResponsesData(commentID: String, parentComment: Post, page: Int, refresh: Bool = false) async -> Bool {
```

Comment:
You have renamed the `getComments` to `getResponsesData` but there are other elements which are according to the old method name like `self.comments`. Do we need to update them as well to it's ok to just update the method name?